github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
+============
+
+**Build Badges**
+
+Short project description. You can use the client's pitch. No more than ten lines.
+
+**Table of Contents**
```

Comment:
Are these bold lines supposed to be comments or subtitles? The "Build Badges" is probably a comment, but this one confused me